lib/request.mjs

```diff
@@ -21,3 +21,9 @@ export const post = (uri, payload, config = {}) => {
   logger.verbose('Payload:', payload);
   return axios.post(uri, payload, config).then(logAndReturn);
 };
+
+export const get = (uri, queryString, config = {}) => {
+  logger.verbose('Sending a GET request to %s', uri);
+  logger.verbose('QueryString:', queryString);
+  return axios.get(`${uri}?${queryString}`, config).then(logAndReturn);
```

Comment:
Might be cool to just call querystring.stringify() on an object, instead of expecting this to be called with a querystring 🤷‍♂️ Future things.

Reply:
Good idea. I've opened an issue to track that: #5